independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/parameters/converters/CharParamConverter.java

```diff
@@ -0,0 +1,26 @@
+package org.jboss.resteasy.reactive.server.core.parameters.converters;
+
+public class CharParamConverter implements ParameterConverter {
+
+    @Override
+    public Object convert(Object parameter) {
+        String str = parameter.toString();
```

Comment:
For `char` it's not clear what a null value is, though I guess the same null value as for `int`? If missing `int` params produce a `0` value, then, probably so should `int` params, no?

Reply:
The value is never `null` so this does not apply

Reply:
Oh, so it's the injector which produces `0` for missing `char` params? OK, fine.